### PR TITLE
[release] merging master-release-update into master

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @GuillaumeGomez @EPashkin @sdroege

### CODE REVIEW

File: atk/vendor.md [hunk 1]
```diff
@@ -336,7 +336,7 @@ height to set for `self`
 # Returns
 
 `true` or `false` whether the size was set or not
-<!-- trait ComponentExt::fn connect_bounds_changed -->
+<!-- trait ComponentExt::fn bounds-changed -->
```
Comment (EPashkin) [inline atk/vendor.md]: IMHO it wrong

Comment (GuillaumeGomez) [reply atk/vendor.md]: What the hell! O.o

Comment (EPashkin) [reply atk/vendor.md]: All signal broken 😢

Comment (EPashkin) [reply atk/vendor.md]: Seems https://github.com/gtk-rs/gir/blob/master/src/codegen/doc/mod.rs#L295 need use `analysis.connect_name` if it possible

Comment (GuillaumeGomez) [reply atk/vendor.md]: Yep... Do you mind doing it? I guess I'll send the documentation PR once fixed and end the release meanwhile.

Comment (EPashkin) [reply atk/vendor.md]: Ok.

Comment (GuillaumeGomez) [reply atk/vendor.md]: Thanks!

**Pull request closed without merging**